testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/deployment/DeploymentScannerShutdownTestCase.java

```diff
@@ -120,22 +110,32 @@ public static void addDeploymentScanner() throws Exception {
 
     @After
     public void cleanAll() throws Exception {
-        removeRules();
-        removeDeploymentScanner(client, scannerName);
-        removeDeploymentScannerExtension();
+        try {
+            removeRules();
+        } catch (Exception e) {
+
+        }
+        try {
+            removeDeploymentScanner(scannerName);
+        } catch (Exception e) {
+
+        }
+        try {
+            removeDeploymentScannerExtension();
+        } catch (Exception e) {
+
+        }
```

Comment:
If one of those exceptions happen, it could be worth storing in a variable and rethrowing after container.stop()?